packages/airnode-examples/README.md

```diff
@@ -152,26 +152,7 @@ Refer to the
 [documentation](https://docs.api3.org/airnode/v0.2/grp-providers/guides/build-an-airnode/configuring-airnode.html) for
 more details.
 
-### 9. Build docker artifacts
```

Comment:
This is not right. There might be some changes in the deployer on master - so you must run it with latest docker (built from source). This is even a bigger issue for airnode-client - since you will be using outdated code.